cpu/stm32l4/include/cpu_conf.h

```diff
@@ -36,7 +36,7 @@ extern "C" {
  * @{
  */
 #define CPU_DEFAULT_IRQ_PRIO            (1U)
-#define CPU_IRQ_NUMOF                   (82U)
+#define CPU_IRQ_NUMOF                   (81U)
```

Comment:
82 is correct, le L432 has 82 interrupts (I started to look at nucleo32-l432 already). The last in CRS global interrupt.